src/content/docs/en/guides/view-transitions.mdx

```diff
@@ -664,14 +664,14 @@ An event that fires at the end of page navigation, after the new page is visible
 
 The `<ViewTransitions />` component fires this event both on initial page navigation for a pre-rendered page and on any subsequent navigation, either forwards or backwards.
 
-You can use this event to run code on every page navigation, or only once ever:
+You can use this event to run code on every page navigation. That is especially useful when it comes to setting up event listeners that would otherwise be lost during navigation.
 
-```astro "{ once: true }"
+```astro
 <script>
   document.addEventListener('astro:page-load', () => {
-    // This only runs once.
-    setupStuff();
-  }, { once: true });
+    // This runs on first page load and after every navigation.
+    setupStuff(); // e.g. add click event listeners
```

Comment:
I guess it doesn’t **have** to be click listeners, so we can simplify a bit here:

```suggestion
    setupStuff(); // e.g. add event listeners
```